data/wp/wp-content/mu-plugins/epfl-functions.php

```diff
@@ -238,8 +238,14 @@ function colored_box( $atts, $content = null ) {
     so it's impossible to switch to the other language
 */
 function http_status_change_to_non_cacheable($status, $location) {
-   /* We only do 303 redirect if redirection are local to website. */
-   return (strpos($location, $_SERVER['SERVER_NAME'])!==false)? 303: $status;
+      /* We update header to avoid caching when using 302 redirect on local host */
+   if($status==302 && strpos($location, $_SERVER['SERVER_NAME'])!==false)
+   {
+        header("Cache-Control: no-store, no-cache, must-revalidate, max-age=0");
+        header("Cache-Control: post-check=0, pre-check=0", false);
```

Comment:
Je sais que j'aurais pu te dire ça avant (pour la PR sur release2018 par ex.), mais tu peux shooter la 245 et la 246 : 
- Pragma: no-cache est un en-tête pour les requêtes (https://tools.ietf.org/html/rfc7234#section-5.4)
- post-check et pre-check sont des en-têtes non-standard d'IE (5) et IIS et sont bidon aussi (https://stackoverflow.com/questions/34663916/are-cache-control-pre-check-and-post-check-headers-still-supported-by-ie)

Voir aussi le blurb sur `options.ignoreCargoCult` dans https://github.com/kornelski/http-cache-semantics ☺

Reply:
Corrigé et j'ai fait #819 pour mettre à niveau ce qu'on a mergé dans "release2018" hier. Je te laisse review aussi.